src/encoded/schemas/image_setting.json

```diff
@@ -0,0 +1,46 @@
+{
+    "title": "Image Settings",
+    "description": "Microscopy metadata specific to the image",
+    "id": "/profiles/image_setting.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["settings"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted" },
+        { "$ref": "mixins.json#/modified" },
+        { "$ref": "mixins.json#/tags" },
+        { "$ref": "user_content.json#/properties" }
+    ],
+    "mixinFacets": [
+        { "$ref": "mixins.json#/facets_common" }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "status": {
+            "title": "Status",
+            "type": "string",
+            "permission": "import_items",
+            "default": "draft",
+            "enum": [
```

Comment:
Good point, but it would make sense to have it consistent with MicroscopeConfiguration, which currently uses `draft` instead of `in review by lab`. In addition, since any user can create these two items and edit them as much as they want, `draft` seems more intuitive. We could later decide to change both and upgrade those schemas.
Most importantly, using `draft` should not impact the functioning of the release script. That script is only used to increase the status number (all statuses above are standard for these items) and `draft` has already the same value (4) as `in review by lab`.